packages/react-dom/src/client/ReactDOM.js

```diff
@@ -188,6 +187,11 @@ if (exposeConcurrentModeAPIs) {
     }
   };
 }
+if (__DEV__) {
+  ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Tests = [
```

Comment:
I could just call this something like `__SECRET_TEST_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED` on `ReactDOM` directly.